torch/csrc/deploy/interpreter/interpreter_impl.h

```diff
@@ -80,6 +80,9 @@ struct Obj {
   Obj callKwargs(std::unordered_map<std::string, c10::IValue> kwargs);
   bool hasattr(const char* attr);
   Obj attr(const char* attr);
+  InterpreterSessionImpl* getInterpreterSessionImpl() {
```

Comment:
We should not expose this object's `InterpreterSessionImpl` as a public part of the interface. My suggestion: since `InterpreterSessionImpl` is already a friend class, write an `InterpreterSessionImpl::isOwner(Obj)` and use that instead.

Reply:
sounds good